kong/core/cluster.lua

```diff
@@ -63,33 +72,35 @@ local function async_autojoin(premature)
     if (autojoin_retries < ASYNC_AUTOJOIN_RETRIES) then
       create_timer(ASYNC_AUTOJOIN_INTERVAL, async_autojoin)
     end
+  elseif err ~= "exists" then
+    log_error(err)
   end
 end
 
 local function send_keepalive(premature)
   if premature then return end
 
-  local lock = resty_lock:new("cluster_locks", {
-    exptime = KEEPALIVE_INTERVAL - 0.001
-  })
-  local elapsed = lock:lock("keepalive")
-  if elapsed and elapsed == 0 then
+  local ok, err = get_lock(KEEPALIVE_KEY, KEEPALIVE_INTERVAL)
+  if ok then
+    log_debug("sending keepalive")
     -- Send keepalive
```

Comment:
Could we get rid of useless comments like this one at the same time? This is obvious + the log added just before states the same as well.